frontend/coprs_frontend/coprs/logic/coprs_logic.py

```diff
@@ -692,8 +694,7 @@ def validate(cls, copr, dirname):
                 f"Please use directory format {copr.name}:custom:<SUFFIX_OF_CHOICE> "
                 f"or {copr.name}:pr:<ID> (for automatically removed directories)"
             )
-
-        if not all(x.isalnum() for x in dirname.split(":")[1:]):
+        if not all(x.isalnum() for x in re.split(r"[:-]+", dirname)[1:]):
```

Comment:
this deserves an in-line note :-) I'm not sure this is correct

We basically only want to allow one more `:`-separated field, right?  I'd wish we had a test-case for the validation.